src/encoded/static/components/item-pages/components/WorkflowNodeElement.js

```diff
@@ -167,16 +167,6 @@ export class WorkflowNodeElement extends React.PureComponent {
         return output; 
     }
 
-    containerStyle(){
```

Comment:
This is in CSS now... except for width : columnWidth which isn't needed as width should be inherited from parent element.